content/1.2/docs/core-concepts/workflow-types/_index.md

```diff
@@ -65,7 +65,7 @@ The *workflowOptions* object must possess six essential attributes with specific
 ```
 
 ##### Examples:
-- [MTA](https://github.com/parodos-dev/serverless-workflows/blob/v1.2.x/mta/mta.sw.yaml)
+- [MTA](https://github.com/parodos-dev/serverless-workflows/blob/v1.2.x/mta-v7.x/mta.sw.yaml)
```

Comment:
https://github.com/parodos-dev/serverless-workflows/blob/v1.2.x/mta/mta.sw.yaml still exists in 1.2.x

All this MTA versions, it's a mess, I think we support v6 for <= 1.2 

But here, what you do is to say that the "good" one to use and our primary choice is v7, right?